src/main/java/me/botsko/prism/Prism.java

```diff
@@ -394,6 +415,22 @@ public void onEnable() {
         });
     }
 
+    private Logger createPrismLogger(){
```

Comment:
Codacy found an issue: [WhitespaceAround: '{' is not preceded with whitespace.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5911179)